docs/astro/sql/operators/dataframe.rst

```diff
@@ -0,0 +1,17 @@
+======================================
+dataframe operator
+======================================
+
+.. _dataframe_operator:
+
+When to use the ``dataframe`` operator
```

Comment:
There should be a mention in this section about how a user can use the `output_table` parameter to turn the resulting dataframe into a table.

Reply:
@dimberman `DataframeOperator` is inheriting from `DecoratedOperator` in the current [code](https://github.com/astronomer/astro-sdk/blob/main/src/astro/sql/operators/dataframe.py#L76). This doesn't have `output_table`.

I feel it should be inherited from `BaseSQLDecoratedOperator` as in [base_decorator](https://github.com/astronomer/astro-sdk/blob/main/src/astro/sql/operators/base_decorator.py#L15). 

WDYT?

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/631/commits/46ed25c9f06bf5a843281352718547b3dc5f47f2) the change @dimberman